### DISCUSSION TITLE
Unclear behavior of downstream / upstream quantity

### DISCUSSION THREAD

**Discussion by BigDanTheOne** [frePPLe/frepple#663] [Q&A]
Hello! I'm new to frepple. Currently, I am having difficulty understanding how downstream and upstream operations work in a manufacturing order (MO). I would greatly appreciate some assistance with this matter. 

I understand that:
- The "downstream operations" allows me to follow how a certain produced quantity is being consumed by other manufacturing and ultimately ends up in a sales order for an end item.
- The "upstream operations" allows me to trace where the consumes material came from that a sales order or manufacturing order.
- Quantity **somehow** reflect the pegging of materials flow between operations. 

What I don't understand is **how exactly**
Take a look at the screenshots:
<img width="832" alt="image" src="https://github.com/frePPLe/frepple/assets/45150310/84a08833-a75e-41e4-85e0-a93630b36304">
<img width="930" alt="image" src="https://github.com/frePPLe/frepple/assets/45150310/75883abb-25c9-4a75-90a3-3d7c733eb33e">

Here we have MO 5 and 18. MO 5 produces multiple items, part of them are passed into MO 18.
Inside "downstream operations" of MO 5 it is written that 2/2 of **something**  2/2  of **something** else are passed into MO 18
Inside "downstream operations" of MO 18 it is written that 2/2 of **something** and 2/2  of **something** else are received in MO 18

My questions are:
1) What are this **something** that are passed into MO? Items? If Items than how can I understand what specific items are being passed?
2) Why do I have a situation where **something** that are passed into MO are not equal to **something** that are received?

**Comment by BigDanTheOne** [comment]
@jdetaeye Hello! I'm sorry to bother you, but you're my last hope for understanding this. I would really appreciate your help.

**Comment by jdetaeye** [comment]
Here are 2 slides from our training material.  It's a straightforward FIFO material allocation across all levels in the supply path.  The upstream&downstream operations just follow this material flow.

![image](https://github.com/user-attachments/assets/a4999a1e-bbba-479d-b785-4832020197cb)

![image](https://github.com/user-attachments/assets/1c4660f2-660f-4461-8d66-c6c87a5238ff)
